### PR TITLE
Created BenefitCard for new benefits page

### PR DESCRIPTION
**Pull request by C-STYR**
- creates BenefitCard
- updates GridDisplay and associated components
- adds associated imagery

### CODE REVIEW

File: pages/about/benefits/index.mdx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+<GridDisplay bg="flatWhite" cardStyle="benefitCard">
+  <BenefitCard title="Dental, Vision, Life and Disability" cardBG="gray" src="./assets/guardian.svg">
+    - Guardian Dental PPO 1500-1500Vision
+    - Guardian-VSP Vision Plan C (Choice)
+  </BenefitCard>
+  <BenefitCard  title="Life Insurance" cardBG="gray" src="./assets/guardian.svg">
+    - [Guardian Guardian Base Life $50,000]()
+  </BenefitCard>
+  <BenefitCard  title="Long Term Disability" cardBG="gray" src="./assets/guardian.svg">
+    - [Guardian Guardian Long-Term Disability (age 65)]()
```
Comment (deliaconstantino) [inline pages/about/benefits/index.mdx:57]: Are there links missing here (lines 54 & 57)?

**Review by deliaconstantino — APPROVED** [review]

File: components/GridDisplay/BenefitCard.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+          borderRadius="0 0 8px 8px"
+        >
+          <Flex
+            flexDirection="column"
+            pt={2}
+            css={css({ "p, li": { fontSize: "text-md", lineHeight: "md" } })}
+          >
+            {children}
+          </Flex>
+        </Flex>
```
Comment (Alqanar) [inline components/GridDisplay/BenefitCard.tsx:102]:
```suggestion
        <Flex
          flexDirection="column"
          flexGrow={1}
          p={4}
          borderRadius="0 0 8px 8px"
          css={css({ "p, li": { fontSize: "text-md", lineHeight: "md" } })}
        >
          {children}
        </Flex>
```

Comment (Alqanar) [reply components/GridDisplay/BenefitCard.tsx:102]: We can get rid of inner wrapper with no visual changes

File: components/GridDisplay/BenefitCard.tsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  title,
+  cardBG,
+}: BenefitCardProps) => {
+  //add additional backgrounds to cardBackgrounds
+  const backgroundImage = cardBackgrounds[cardBG];
+
+  return (
+    <Flex flexDirection="column">
+      <Box
+        fontSize="text-xl"
```
Comment (Alqanar) [inline components/GridDisplay/BenefitCard.tsx:50]: Please add a tag here `as="h2"`. 
In my opinion, since we have a header, let it have an appropriate tag

**Review by Alqanar — CHANGES_REQUESTED** [review]
Thank you for the PR!
I left few comments.

File: pages/about/benefits/index.mdx (hunk 1 above)
Comment (C-STYR) [reply pages/about/benefits/index.mdx:57]: Yes, they have not been provided by Design yet.

File: components/GridDisplay/BenefitCard.tsx [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+            {src && (
+              <NextImage
+                src={src}
+                alt={title ? title + " provider" : "colorful background"}
+                height={40}
+                width={282}
+              />
+            )}
+          </Flex>
+        </Flex>
```
[the remark this answers is not in the document]
Comment (C-STYR) [reply components/GridDisplay/BenefitCard.tsx:84]: Good, very efficient, thanks.

File: components/GridDisplay/BenefitCard.tsx [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+  css({
+    width: ["340px", "auto", "auto"],
+    height: "100%",
+    flexDirection: "column",
+    borderRadius: "md",
+    textDecoration: "none",
+    boxShadow: "0 1px 4px rgba(0, 0, 0, 0.24)",
+    bg: "white",
+  })
+);
```
[the remark this answers is not in the document]
Comment (C-STYR) [reply components/GridDisplay/BenefitCard.tsx:118]: Fixed.

File: components/GridDisplay/BenefitCard.tsx [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+  return (
+    <Flex flexDirection="column">
+      <Box
+        fontSize="text-xl"
+        lineHeight="lg"
+        color="dark-purple"
+        fontWeight="black"
+        minHeight={["8px", "32px"]}
+        mb={4}
```
[the remark this answers is not in the document]
Comment (C-STYR) [reply components/GridDisplay/BenefitCard.tsx:55]: @Alqanar the `min-height` property was actually for vertical alignment of the top of the card.  If removed, we get this: 

Screen Shot 2021-11-19 at 3 14 25 PM

...when there is no `title` passed in, as the `title` is part of the card though properly outside its visible border/box-shadow area.  `min-height` was the method I chose to retain that negative space throughout responsive views, as my experimentations with `mt` were unsatisfactory if `title` were present.

File: components/GridDisplay/BenefitCard.tsx (hunk 3 above)
Comment (C-STYR) [reply components/GridDisplay/BenefitCard.tsx:50]: Agreed.

File: components/GridDisplay/BenefitCard.tsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (Alqanar) [reply components/GridDisplay/BenefitCard.tsx:55]: That's weird. Let me see.

[the remark this answers is not in the document]
Comment (Alqanar) [reply components/GridDisplay/BenefitCard.tsx:55]: I propose this option
```      
      {!!title && (
        <Box
          as="h2"
          fontSize="text-xl"
          lineHeight="lg"
          color="dark-purple"
          fontWeight="black"
          mt={[3, 0]}
        >
          {title}
        </Box>
      )}

      <StyledWrapper
        backgroundImage={backgroundImage}
        pt={src ? 0 : ["90px", "112px"]}
        mt={title ? [3, 4] : [5, 8]}
      >
        {/* top half */}
```
would look something like this

Screenshot 2021-11-19 at 17 40 10
Screenshot 2021-11-19 at 17 39 53

[the remark this answers is not in the document]
Comment (Alqanar) [reply components/GridDisplay/BenefitCard.tsx:55]: In this option we put the title under the condition. `mt` moved to the next tag (`StyledWrapper`). And If we don't have the title we will increase margin.

[the remark this answers is not in the document]
Comment (C-STYR) [reply components/GridDisplay/BenefitCard.tsx:55]: This works but creates greater complexity.  Why is `min-height` objectionable?  Is there a technical reason?

[the remark this answers is not in the document]
Comment (Alqanar) [reply components/GridDisplay/BenefitCard.tsx:55]: The problem is that you are using the `min-height` property for margin, which means you are not using it as intended.

**Review by Alqanar — APPROVED** [review]

**Pull request merged**